### DISCUSSION TITLE
Massupdate of Float Fields With Units and Cached Files

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#796] [Q&A] [answered]
After the Update to 1.6, Product Managers experienced missing units:

![image](https://github.com/atrocore/atrocore/assets/1337223/fc13b398-f438-4559-ac33-7ebb4cd8e002)

I told them several ways how to empty their browser cache, but the problem persisted and worse, when they saved values for temperature, the unit was lost.

![image](https://github.com/atrocore/atrocore/assets/1337223/799b7096-e7b8-450f-8752-654344d4867e)

With the listing of ProductAttributeValue I get a nice overview, but I can't see a way to mass update all units and keep the value intact.

My question is:

1. Did I overlook something? Would I be able to export/import only the values? (I found only how to export all attribute values, but not with the unit)
2. How to prevent the caching issue at all?

If I assume they did empty their browser cache (and tried it in incognito mode) and they still see the garbled units, then there must be a proxy cache/firewall caching some JavaScript? Is there a way to prevent that (http header[1])? I'm sure you have some sort of cache busting mechanism in AtroCore, but how to make sure after an update, all the fresh files are requested, without performing some Cacheclearing Voodoo?

PS: I didn't experience any of this, and if I see irregularities, I know how to make sure the cache is clean, and I'm on a different network anyway. But the product managers shouldn#t experience any of that.

[1] you are already using `no-store, no-cache, must-revalidate, post-check=0, pre-check=0` that should prevent any caching, but what else could it be?

**Comment by rratsun** [comment, marked as answer]
In the latest version of Core and PIM it should not be. Which version do you have in your installation? As for export and import, yes you had problem when user try to export Attribute Value, because we hided valueUnit field (it is a mistake, this field should be). Generally, users should be able to import and export value and valueUnit separately. If it is not so, so it will be so in the nearest patches.
